napalm/eos/eos.py

```diff
@@ -85,7 +87,16 @@ class EOSDriver(NetworkDriver):
         re.VERBOSE,
     )
 
-    def __init__(self, hostname, username, password, timeout=60, optional_args=None):
+    def __init__(
+        self,
+        hostname,
+        username,
+        password,
+        timeout=60,
+        rosetta_driver=None,
+        yang_model="openconfig",
```

Comment:
we could also set this to None by default and avoid loading the rosetta driver, that'd save some memory

Reply:
Ah, only now seeing this comment. Left a comment around this line on the base driver. I think that'd make sense, and somewhat related to my suggestion.